Jenkinsfile

```diff
@@ -851,6 +851,38 @@ pipeline {
                         */
                     }
                 }
+                stage ('IOF-build') {
+                    agent {
+                        label 'ci_vm1'
```

Comment:
Why not build in a docker container?

Reply:
Because the path might be incorrect, after the 1st build only using a VM and ssh is possible as you need to nfs trick to get the right directory.

Reply:
Hrm.  Yeah.  I can see why that would be.

Not asking you to experiment with it explicitly, but some symlink trickery might solve it in a container perhaps.